.github/workflows/appimage.yml

```diff
@@ -37,7 +37,7 @@ jobs:
       options: --platform ${{ matrix.platform }}
     timeout-minutes: 180
     outputs:
-      date: ${{ steps.build_test_app.outputs.date }}
+      date: ${{ steps.build_appimage.outputs.date }}
```

Comment:
Where is this output being used? I can't seem to find it, but maybe I'm missing something.

Reply:
I don't have an answer for this, to be honest. We changed this because we added the action linter and had to change `build_test_app` to `build_appimage` in order for the linter to be happy. While we were at it, we added `echo "date=`date +%F`" >> $GITHUB_OUTPUT` because that's what is being done in RDK, but we did wonder if we needed `date` altogether.

Given that you think it's unused and should be removed, I will do that.

Reply:
Yeah, if there's not a job elsewhere that depends on this job (AND actually uses that output) then it can go away and simplify the file.

In RDK, it's because a later job actually tests the built appimage, and needs the date to get the right URL for the temporary file (since computer clocks might not be in the same timezone, or tests might happen near midnight.)